app/views/ppd/index.html.haml

```diff
@@ -1,5 +1,7 @@
 -# frozen_string_literal: true
 
+= content_for(:title, "Search the price paid dataset".html_safe)
```

Comment:
this also doesn't need `.html_safe` as there's nothing to make safe

Reply:
:+1: